### PR TITLE
feat(ForumChannel): return message when creating thread

### PR DESCRIPTION
**Pull request by Victorsitou**
## Summary

https://github.com/discord/discord-api-docs/pull/4850

~~PR 2.0 coming soon!!1! this is 1.5~~

## Checklist

- [x] If code changes were made, then they have been tested
    - [x] I have updated the documentation to reflect the changes
    - [x] I have formatted the code properly by running `task lint`
    - [x] I have type-checked the code by running `task pyright`
- [ ] This PR fixes an issue
- [x] This PR adds something new (e.g. new method or parameters)
- [x] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by shiftinv — APPROVED** [review]
nice, thanks!

File: disnake/http.py [hunk 1]
```diff
@@ -1202,24 +1202,20 @@ def start_thread_in_forum_channel(
...
-    ) -> Response[threads.Thread]:
-        valid_keys = (
-            # Thread fields
-            "name",
-            "auto_archive_duration",
-            "rate_limit_per_user",
-            "type",
-            # Message fields
+    ) -> Response[threads.ForumThread]:
+        valid_thread_keys = {"name", "auto_archive_duration", "rate_limit_per_user", "type"}
```
Comment (onerandomusername) [inline disnake/http.py:1206]: this should continue to remain a tuple as all other valid_keys

File: disnake/http.py [hunk 2]
```diff
@@ -1202,24 +1202,20 @@ def start_thread_in_forum_channel(
...
+        valid_message_keys = {
```
Comment (onerandomusername) [inline disnake/http.py:1207]: also should be a tuple

**Review by onerandomusername — CHANGES_REQUESTED** [review]

File: disnake/http.py (hunk 1 above)
Comment (onerandomusername) [reply disnake/http.py]: https://github.com/DisnakeDev/disnake/pull/504/commits/e827c5209f77a8930809a3f18d0f2436291a74eb

File: disnake/http.py (hunk 2 above)
Comment (onerandomusername) [reply disnake/http.py]: https://github.com/DisnakeDev/disnake/pull/504/commits/e827c5209f77a8930809a3f18d0f2436291a74eb

**Review by onerandomusername — APPROVED** [review]
lgtm

**Pull request merged**